<launch>
  <node pkg="corner_detect" type="lasescan" name="laser_scan" output="log" launch-prefix="xterm -e gdb --args"/>

</launch>
